tests/test_table_paste_col.py

```diff
@@ -0,0 +1,45 @@
+import sys
+import unittest
+from io import StringIO
+from tabletools import table_paste_col
+
+class TestTablePasteCol(unittest.TestCase):
+    def test_parser(self):
+        parser = table_paste_col.parse_args(['--table', 'foo', '--col-name', 
+            'bar', '--col-value', 'baz', '--at-end'])
+        self.assertEqual(parser.table, 'foo')
+        self.assertEqual(parser.col_name, 'bar')
+        self.assertEqual(parser.col_value, 'baz')
+        self.assertTrue(parser.at_end)
+
+        parser = table_paste_col.parse_args(['--table', 'foo', '--col-name', 
+            'bar', '--col-value', 'baz'])
+        self.assertEqual(parser.table, 'foo')
+        self.assertEqual(parser.col_name, 'bar')
+        self.assertEqual(parser.col_value, 'baz')
+        self.assertFalse(parser.at_end)
+        
+    def test_add_col(self):
+        parser = table_paste_col.parse_args(['--table', 'table',
+            '--col-name', 'foo', '--col-value', 'bar'])
+        table = StringIO("A\tB\tC\nD\tE\tF\nG\tH\tI\n")
+        expected = "foo\tA\tB\tC\nbar\tD\tE\tF\nbar\tG\tH\tI\n"       
+        out = StringIO()
+        table_paste_col.add_col(table, parser.separator, parser.col_name, 
+                parser.col_value, parser.at_end, out)
```

Comment:
fyi you don't need to use/test the parser again. You have already done this in the previous method and you can simplify the code here. This way you also isolate what each method is doing and when it fails you will know where to look. (The same comment applies to the next method.

```python
        table = StringIO("A\tB\tC\nD\tE\tF\nG\tH\tI\n")
        expected = "foo\tA\tB\tC\nbar\tD\tE\tF\nbar\tG\tH\tI\n"       
        out = StringIO()
        table_paste_col.add_col(table, "\t", "foo", "bar", False, out)